app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -128,10 +141,12 @@ class MusicsListFragment : Fragment() {
     }
 
     private fun musicListener(music: Music) {
-        val action = MusicsListFragmentDirections.actionMusicsListFragmentToPlayerFragment(music)
-
-        findNavController().navigate(action)
+//        val action = MusicsListFragmentDirections.actionMusicsListFragmentToPlayerFragment(music)
+//
+//        findNavController().navigate(action)
+        ExoPlayerService.actionStart(context, musicList,music.id)
```

Comment:
in testi inja goazshte shode?!
chera jeloye baz shodane player fragment ro greftim?

Reply:
are testi gozashte shode

Reply:
Are vase test gozashtam